MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+* k8s.node.filesystem.usage
+* k8s.node.memory.available
+* k8s.node.memory.major_page_faults
+* k8s.node.memory.page_faults
+* k8s.node.memory.rss
+* k8s.node.memory.usage
+* k8s.node.memory.working_set
+* k8s.node.network.errors
+* k8s.node.network.io
+* k8s.pod.cpu.time
+* k8s.pod.cpu.utilization
+* k8s.pod.filesystem.available
+* k8s.pod.filesystem.capacity
+* k8s.pod.filesystem.usage
+* k8s.pod.memory.available
+* k8s.pod.memory.major_page_faults
+* k8s.pod.memory.page_faults
+* k8s.pod.memory.rss
+* k8s.pod.memory.usage
+* k8s.pod.memory.working_set
+* k8s.pod.network.errors
+* k8s.pod.network.io
+* k8s.pod.phase
+* k8s.replicaset.available
+* k8s.replicaset.desired
+* otelcol_exporter_queue_size
+* otelcol_exporter_send_failed_log_records
+* otelcol_exporter_send_failed_metric_points
+* otelcol_exporter_sent_log_records
+* otelcol_exporter_sent_metric_points
+* otelcol_otelsvc_k8s_ip_lookup_miss
+* otelcol_otelsvc_k8s_namespace_added
+* otelcol_otelsvc_k8s_namespace_updated
+* otelcol_otelsvc_k8s_pod_added
+* otelcol_otelsvc_k8s_pod_table_size
+* otelcol_otelsvc_k8s_pod_updated
+* otelcol_process_cpu_seconds
+* otelcol_process_memory_rss
+* otelcol_process_runtime_heap_alloc_bytes
+* otelcol_process_runtime_total_alloc_bytes
+* otelcol_process_runtime_total_sys_memory_bytes
+* otelcol_process_uptime
+* otelcol_processor_accepted_log_records
+* otelcol_processor_accepted_metric_points
+* otelcol_processor_batch_batch_send_size_bucket
+* otelcol_processor_batch_batch_send_size_count
+* otelcol_processor_batch_batch_send_size_sum
+* otelcol_processor_batch_timeout_trigger_send
+* otelcol_processor_dropped_log_records
+* otelcol_processor_dropped_metric_points
+* otelcol_processor_refused_log_records
+* otelcol_processor_refused_metric_points
+* otelcol_receiver_accepted_metric_points
+* otelcol_receiver_refused_metric_points
+* otelcol_scraper_errored_metric_points
+* otelcol_scraper_scraped_metric_points
+* scrape_duration_seconds
+* scrape_samples_post_metric_relabeling
+* scrape_samples_scraped
+* scrape_series_added
+* system.cpu.load_average.15m
+* system.cpu.load_average.1m
+* system.cpu.load_average.5m
+* system.cpu.time
+* system.disk.io
+* system.disk.io_time
+* system.disk.merged
+* system.disk.operation_time
+* system.disk.operations
+* system.disk.pending_operations
+* system.disk.weighted_io_time
+* system.filesystem.inodes.usage
+* system.filesystem.usage
+* system.memory.usage
+* system.network.connections
+* system.network.dropped
+* system.network.errors
+* system.network.io
+* system.network.packets
+* system.paging.faults
+* system.paging.operations
+* system.paging.usage
+* system.processes.count
+* system.processes.created
+* up
+
+### Changes in Objects in Splunk OpenTelemetry Connector for Kubernetes (SCK-OTEL)
+This chart currently does not support the collection of Kubernetes objects. This is future functionality. You can still use SCK to collect other objects with current functionality.
+
+## Migration Overview
```

Comment:
"Migration overview"

"Options"

"The following table shows the options for migrating ..."